JSDemos/Demos/Popup/Scrolling/jQuery/styles.css

```diff
@@ -1,13 +1,44 @@
-#textBlock {
-  padding-left: 20px;
-  padding-right: 20px;
-  line-height: 1.6em;
-}
-
-.center {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  padding-top: 20px;
-  width: 300px;
+.label {
+  color: rgba(51, 51, 51, 0.6);
+  font-size: 12px;
+}
+
+.demo-container {
+  height: 450px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  gap: 40px;
+}
+
+.button-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  gap: 15px;
+}
+
+.dx-popup-content {
+  font-size: 12px;
+}
+
+.dx-popup-title .dx-toolbar-before {
```

Comment:
Why do we need this style?
It looks like we're working around some internal restriction

Reply:
There was some reason to add this css rule. It's not require anymore and I removed it.